packages/workbox-range-requests/Plugin.mjs

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import {createPartialResponse} from './createPartialResponse.mjs';
+
+import './_version.mjs';
+
+/**
+ * A class implementing the `cachedResponseWillBeUsed` lifecycle callback.
```

Comment:
This description should cover the purpose of the class, not the implementation details.

> The range request plugin makes it easy for a Request with a 'range' header to be forfilled by a cached response.
> It does this by intercepting the `cachedResponseWillBeUsed` plugin callback and returning the appropriate subset of the cached Response body.

Reply:
Done.